packages/now-cli/src/util/output/create-output.ts

```diff
@@ -52,6 +55,7 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
             right: 1,
           },
           borderColor: 'yellow',
+          ...options?.boxen,
```

Comment:
Can we remove the "workaround" from #3439 by updating `boxen`?

Reply:
Seems like it was already using the latest version, so I just removed the workaround.